src/main/java/pb360/data/repository/MessageObjectRepository.java

```diff
@@ -0,0 +1,22 @@
+package pb360.data.repository;
+
+
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+//import org.springframework.data.repository.CrudRepository;
+
+import pb360.model.*;
```

Comment:
should not import all entries, please only import what entries we used

Reply:
i changed pb360.model.* to pb630.model.MessageObject